src/targets/gpu/jit/ck_gemm_softmax_gemm.cpp

```diff
@@ -222,7 +228,7 @@ struct ck_gemm_softmax_gemm_compiler : compiler<ck_gemm_softmax_gemm_compiler>
         tuning_config tc;
         auto shapes    = to_shapes(ins->inputs());
         auto problem   = create_problem(shapes, create_settings(ins, op));
-        auto solutions = problem.GetSolutions(ctx.get_current_device().get_gfx_name());
+        auto solutions = problem.GetSolutions(ctx.get_current_device().get_gfx_name(), "", "");
```

Comment:
We shouldn't need those extra parameters, Ck should be updated to make them defaulted.

Reply:
I created a PR on CK: https://github.com/ROCm/composable_kernel/pull/2020